### PR TITLE
4.4.15: demuxer timeshift improvements

### PR DESCRIPTION
**Pull request by ksooo**
Requires Kodi fix: https://github.com/xbmc/xbmc/pull/14488

Code taken from https://github.com/FernetMenta/pvr.hts/commit/9999beb87dab709689384f1fe03b65e8f2a01a63 Details discussed there.

### CODE REVIEW

**Comment by ksooo** [comment]
Will runtime-test tonight...

File: src/tvheadend/HTSPDemuxer.cpp [hunk 1]
```diff
@@ -297,7 +297,13 @@ bool HTSPDemuxer::IsTimeShifting() const
 
 bool HTSPDemuxer::IsRealTimeStream() const
 {
-  return m_subscription.IsActive();
+  if (!m_subscription.IsActive())
+    return false;
+
+  if (m_subscription.GetSpeed() != SPEED_NORMAL)
```
Comment (FernetMenta) [inline src/tvheadend/HTSPDemuxer.cpp]: this is wrong. where did you get this from?

Comment (ksooo) [reply src/tvheadend/HTSPDemuxer.cpp]: Ooops, this was there in the code before the  last change of this method and slipped in accidentally. I will remove that.

**Comment by ksooo** [comment]
@FernetMenta I removed the code piece that slipped in accidentally.

**Pull request merged**